gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.10.2-bin.zip
```

Comment:
NIT:  If would be awesome if we could keep the (new) CI in sync with this.  In `.github/workflows/reusable-workflow.yaml` there is a line that specifies it: `gradle-version: 8.7` that we should bump to `8.10.2`.  

It shouldn't fail without being updated, but each run will be slower.